package.json

```diff
@@ -37,8 +37,8 @@
   "devDependencies": {
     "@types/jest": "^22.0.1",
     "jest": "^22.0.5",
-    "rxjs": "^5.5.0",
+    "rxjs": "^6.0.0",
```

Comment:
I think we can use 6.2.0

Reply:
We can, but the feature is already ready since 6.0.0 I think, but it doesn't matter :)